src/directives/resource/extract.ts

```diff
@@ -47,6 +47,9 @@ export class DirectiveExtract extends Directive {
 		if (this.colony.level < 6) {
 			log.notify(`Removing extraction directive in ${this.pos.roomName}: room RCL insufficient.`);
 			this.remove();
+		}  else if (!this.colony.terminal) {
```

Comment:
good catch; will add condition in `Colony.ts` to not place directive unless terminal is present

Reply:
Added